lib/matplotlib/backends/backend_pdf.py

```diff
@@ -990,13 +990,18 @@ def _write_afm_font(self, filename):
         self.writeObject(fontdictObject, fontdict)
         return fontdictObject
 
-    def _embedTeXFont(self, fontinfo):
-        _log.debug('Embedding TeX font %s - fontinfo=%s',
-                   fontinfo.dvifont.texname, fontinfo.__dict__)
+    def _embedTeXFont(self, dvifont):
+        psfont = dviread.PsfontsMap(dviread.find_tex_file('pdftex.map'))[
+            dvifont.texname]
```

Comment:
Might as well keep the separate variable if this is going to wrap to two lines anyway.
```suggestion
        tex_font_map = dviread.PsfontsMap(dviread.find_tex_file('pdftex.map'))
        psfont = tex_font_map[dvifont.texname]
```